src/content/deployment/_index.en.md

```diff
@@ -5,50 +5,47 @@ weight: 10
+
+In addition to Operator and subctl, Submariner also provides [Helm Charts](helm).
 
 ## Installing subctl
 
 {{< subctl-install >}}
 
-## Deployment of broker
+## Deployment of Broker
+
+The Broker is a set of Custom Resource Definitions (CRDs) backed by the Kubernetes datastore. The Broker should be deployed on a cluster whose Kubernetes API must be accessible by all of the participating clusters:
 
-Please remember, this cluster's API should be accessible from all other clusters:
 ```bash
 subctl deploy-broker --kubeconfig <PATH-TO-KUBECONFIG-BROKER> 
 ```
 
-this will create:
+This will create:
 
-* The submariner-k8s-broker namespace
-* The Cluster and Endpoint (.submariner.io) CRDs in the cluster
-* A service account in the namespace for subsequent subctl access.
+* The `submariner-k8s-broker` namespace.
+* The `Endpoint` and `Cluster` CRDs in the cluster.
+* A Service Account (SA) in the namespace for subsequent subctl access.
 
-And generate a broker-info.subm file which contains the following elements:
+And generate the `broker-info.subm` file which contains the following elements:
 
-* The API endpoint
-* A CA certificate to for the API endpoint
-* The service account token for accessing the API endpoint / submariner-k8s-broker namespace.
+* The API endpoint.
+* A CA certificate for the API endpoint.
+* The Service Account token for accessing the API endpoint.
 * A random IPsec PSK which will be stored only in this file.
-* Globalnet settings
-* Service discovery settings
+* Service Discovery settings.
 
 
 {{% notice info %}}
-This cluster can also participate in the dataplane connectivity with the other clusters, but it will need to be joined (see following step)
+The cluster in which the Broker is deployed can also participate in the dataplane connectivity with other clusters, but it will need to be joined (see following step).
 {{% /notice %}}
 
 ## Joining clusters
 
 
-For each cluster you want to join:
+For each cluster you want to join, issue the following command:
 ```bash
-subctl join --kubeconfig <PATH-TO-JOINING-CLUSTER> broker-info.subm
+subctl join --kubeconfig <PATH-TO-JOINING-CLUSTER> broker-info.subm --clusterid <ID>
 ```
 
-subctl will discover as much as it can, and ask you for any necessary detail it can't figure out like the cluster ID which has to be different between all clusters.
-
-
-## Discovery
-
-Service discovery (via DNS and lighthouse project) is an experimental feature (developer preview), and the instructions to deploy with discovery can be found [here](with-discovery/)
+subctl will automatically discover as much as it can, and prompt the user for any missing necessary information. Note that each cluster must be configured with a unique 'clusterid' value.
```

Comment:
```suggestion
subctl will automatically discover as much as it can, and prompt the user for any missing necessary information. Note that each cluster must be configured with a unique cluster ID.
```